sw/otbn/code-snippets/randomness.s

```diff
@@ -4,10 +4,19 @@
 
 /* Test access to randomness from OTBN. */
 
+.equ CSR_FG0,          0x7c0
```

Comment:
Huh, neat! Despite writing the frontend, I didn't realise that `otbn-as` would allow that! :-D

Reply:
@jadephilipoom we should make sure to recall for the style guide later, to be consistent.